boards/Kconfig

```diff
@@ -1364,13 +1364,21 @@ config ARCH_BOARD_SABRE_6QUAD
 		board featuring the iMX 6Quad CPU.
 
 config ARCH_BOARD_SAMA5D2_XULT
-	bool "Atmel SAMA5D2 Xplained Ultra development board"
+bool "Atmel SAMA5D2 Xplained Ultra development board"
+depends on ARCH_CHIP_ATSAMA5D27
+        select ARCH_HAVE_LEDS
+        select ARCH_HAVE_BUTTONS
+        select ARCH_HAVE_IRQBUTTONS
+---help---
+The port of NuttX to the Atmel SAMA5D2-XULT development board
+
+config ARCH_BOARD_GIANT_BOARD
+	bool "Groboards Giant Board (SAMA5D27D1G)"
 	depends on ARCH_CHIP_ATSAMA5D27
 	select ARCH_HAVE_LEDS
-	select ARCH_HAVE_BUTTONS
-	select ARCH_HAVE_IRQBUTTONS
 	---help---
-		The port of NuttX to the Atmel SAMA5D2-XULT development board
+	    This options selects support for NuttX on the Groboards Giant Board,
+	    featuring the SAMA5D27D1G SIP with 128KB DDR3 RAM.
```

Comment:
I tried to google for the mentioned PN (SAMA5D27D1G) and it returned zero occurrences.
The correct PN for the SiP is ATSAMA5D27C-D1G.

Reply:
@ghn-certi Will fix.